docs/section18/Rule_18-2.md

```diff
@@ -0,0 +1,87 @@
+					- for a multi-zone system, in a building without lab zones, the system can pass if:
+						1. all zones are on the same floor
+						2. there are no other systems of the same type that serve zones on the same floor
+					- - get the list of zones on the same floor by using the function get_zones_on_same_floor: `zones_on_floor = get_zones_on_same_floor(B_RMI,zones_served_by_system[0])`
+					- check if all the zones served by the system are on the same floor: `if all(zone in zones_on_floor for zone in zones_served_by_system):`
+						- now check if there are any other systems of the same system type that serve zones on this floor.  First loop through baseline_system_types_dict again: `for hvac_system_type2 in baseline_system_types_dict:`
+							- check if the hvac_system_type2 is the same type: `if baseline_system_type_compare(hvac_system_type2,system_type,FALSE):`
+								- look through all of the hvacs of this system type: `for hvac_system2_id in baseline_system_types_dict[hvac_system_type2]:`
+									- make sure that hvac_system2_id is not the same as hvac_system_id: `if hvac_system2_id != hvac_system_id:`
+										- now check if hvac_system2 serves any of the zones on the same floor as hvac_system.  Get the zones served by hvac_system2: `zones_served_by_system2 = zones_and_terminal_unit_list_dict[hvac_system2_id]["ZONE_LIST"]`
+										- use set.intersection to see if any of these zones are on the same floor: `if(len(set(zones_served_by_system2).intersection(zones_on_floor))) > 0:`
+											- the system fails, UNLESS system_type is SYS_5 or SYS_7 AND all of the zones in hvac_system2 are lab zones.  Set the result to fail, we'll check these special cases lower down and change the result back to pass if it passes: `result = FAIL`
+											- check if the system type is 5 or 7: `if any(system_type == sys_type for sys_type in [SYS_5,SYS_7]):`
+												- check if all of the zones served by hvac_system are lab zones: `if set(laboratory_zones_meeting_G3_1_1d) == set(zones_served_by_system):`
+													- check if the building has greater than 15,000 cfm lab exhaust.  If so, set result to PASS.  If not, set result to UNDETERMINED: `if lab_zone_exhaust > 15000: result = TRUE`
+													- else set result to UNDETERMINED: `result = UNDETERMINED`
+												- else check if all of the zones served by hvac_system2 are lab zones: `if set(laboratory_zones_meeting_G3_1_1d) == set(zones_served_by_system2):`
+													- check if the building has greater than 15,000 cfm lab exhaust.  If so, set result to PASS.  If not, set result to UNDETERMINED: `if lab_zone_exhaust > 15000: result = TRUE`
+													- else set result to UNDETERMINED: `result = UNDETERMINED`
+						- at this point, there is still one case we haven't handled - a system that is not type 5 or 7 and incorrectly serves a lab type zone.  This could be the case if the result is currently PASS: `if result == PASS:`
```

Comment:
Wouldn't this scenario be covered by 18-1?

Reply:
18-1 covers the correct system type for lab zones.  But lab zones on multiple floors must be served a multi-zone system (5 or 7).